data/bathymetry/preprocess_bathymetry.py

```diff
@@ -46,123 +46,144 @@ def preprocessing_ehydro(tif, bathy_bounds, survey_gdb, output, min_depth_thresh
             Highest value allowed for the minimum depth of the survey
             in feet (Checks for data referenced to a datum).
 
+        Example usage:
```

Comment:
I think this usage might be in the wrong place. Here it's in the `preprocessing_ehydro()` function block.

Also, it looks like it's supposed to run inside a container (`/foss_fim`) but the example paths appear to be external (`/fim-home`).

Reply:
Thanks for the suggestion- I will change its location.
My container is a little bit different for this specific case. Since both input and output data for EHydro bathymetry are in `fim-home`, I mounted them this way.